web/src/lib/domain-logic/getContentWithCardCount.ts

```diff
@@ -0,0 +1,30 @@
+import { filterCertifications } from "@/lib/domain-logic/filterCertifications";
+import { filterFundings } from "@/lib/domain-logic/filterFundings";
+import { getVisibleCertifications } from "@/lib/domain-logic/getVisibleCertifications";
+import { getVisibleFundings } from "@/lib/domain-logic/getVisibleFundings";
+import { Certification, Funding } from "@/lib/types/types";
+import { LookupOperatingPhaseById } from "@businessnjgovnavigator/shared/operatingPhase";
+import { Business } from "@businessnjgovnavigator/shared/userData";
+
+export const getContentWithCardCount = (
+  business: Business | undefined,
+  certifications: Certification[],
+  fundings: Funding[]
+): number => {
+  let count = 0;
+  if (LookupOperatingPhaseById(business?.profileData.operatingPhase).displayCertifications) {
+    count += getVisibleCertifications(
+      filterCertifications(certifications, business as Business),
+      business as Business
+    ).length;
+  }
+
+  if (LookupOperatingPhaseById(business?.profileData.operatingPhase).displayFundings) {
+    count += getVisibleFundings(filterFundings(fundings, business as Business), business as Business).length;
+  }
+
+  if (business?.preferences.visibleSidebarCards.length) {
+    count += business?.preferences.visibleSidebarCards.length;
+  }
+  return count;
+};
```

Comment:
[boulder] I think this should have unit tests

Reply:
wrote a unit test for each individual count and error case